packages/discord.js/typings/index.d.ts

```diff
@@ -4733,14 +4736,14 @@ export type MessageChannelComponentCollectorOptions<T extends MessageComponentIn
 export interface MessageEditOptions {
   attachments?: MessageAttachment[];
   content?: string | null;
-  embeds?: (Embed | APIEmbed)[] | null;
+  embeds?: (JSONEncodable<APIEmbed> | APIEmbed)[] | null;
   files?: (FileOptions | BufferResolvable | Stream | MessageAttachment)[];
   flags?: BitFieldResolvable<MessageFlagsString, number>;
   allowedMentions?: MessageMentionOptions;
   components?: (
     | JSONEncodable<APIActionRowComponent<APIMessageActionRowComponent>>
-    | ActionRow<MessageActionRowComponent>
-    | (Required<BaseComponentData> & ActionRowData<MessageActionRowComponentData | MessageActionRowComponent>)
+    | ActionRowBuilder<MessageActionRowBuilder>
```

Comment:
do you mean MessageActionRowComponent?

Reply:
These types are supposed to accept ActionRow builders with MessageActionRowComponentBuilders. Gonna be honest I got a bit confused with the names but I think the next commit should fix all of this. Please re-review if it doesn't